modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/frontend/data/set/view/table/NotificationQueueEntryTableFDSView.java

```diff
@@ -40,6 +42,22 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		FDSTableSchemaBuilder fdsTableSchemaBuilder =
 			_fdsTableSchemaBuilderFactory.create();
 
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-159052"))) {
+			return fdsTableSchemaBuilder.add(
+				"subject", "subject"
+			).add(
+				"triggerBy", "trigger-by"
+			).add(
+				"fromName", "from"
+			).add(
+				"toName", "to"
+			).add(
+				"status", "status",
+				fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
+					"notificationQueueEntryStatusDataRenderer")
+			).build();
+		}
```

Comment:
You don't need to wrap this entire thing with the feature flag. You can do something like this:
```suggestion
               fdsTableSchemaBuilder.add(
				"subject", "subject"
			).add(
				"triggerBy", "trigger-by"
			).add(
				"fromName", "from"
			).add(
				"toName", "to"
			);
			
		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-159052"))) {
			fdsTableSchemaBuilder.add(
				"status", "status",
				fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
					"notificationQueueEntryStatusDataRenderer")
			);
		}
		else {
		        fdsTableSchemaBuilder.add(
				"sent", "status",
			         fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
				        "notificationQueueEntryStatusDataRenderer");
			);
		}
		
		return fdsTableSchemaBuilder.build();
```